.eslintrc.yml

```diff
@@ -54,6 +54,9 @@ rules:
     - 2
     - "never"
 
+  react/prop-types:
```

Comment:
No clue why this started popping up as I worked on this change, but given that we're primarily using TypeScript over prop types I think it's fine to just turn this off.